src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor.Types.cs

```diff
@@ -23,20 +23,23 @@ private void VisitTypeSymbolWithAnnotations(TypeSymbolWithAnnotations type, Abst
             else
             {
                 typeSymbol.Accept(visitor);
-                switch (isNullable)
+                if (format.CompilerInternalOptions.IncludesOption(SymbolDisplayCompilerInternalOptions.IncludeNonNullableTypeModifier))
```

Comment:
Does this change mean `?` annotations are no longer added for reference types unless the internal option is set?

Reply:
I'll revert this.